### ISSUE TITLE
Add docs on how to use `get-schema`

### ISSUE THREAD

**Issue by schickling** [Urigo/graphql-cli#8]

**Comment by ardatan** [comment]
Currently we have new structure for GraphQL CLI and in the new alpha version `4.0.0-alpha.0` we suggest to use GraphQL Code Generator plugin of GraphQL CLI with Codegen's `schema-ast` plugin. Checkout new instructions and we will be happy to hear your feedback. Feel free to contact us using our Discord channel;
https://discord.gg/xud7bH

**Comment by Urigo** [comment]
We've just released a new version and an alpha release channel for GraphQL CLI - `4.0.0-alpha.XXX`!

We've changed the `get-schema` command to use [GraphQL Code Generator](http://graphql-code-generator.com) plugin of GraphQL CLI with the Codegen's [`schema-ast` plugin](https://graphql-code-generator.com/docs/plugins/schema-ast). 

I hope the documentation is now more understandable. 

Checkout new instructions and the [migration guide](https://github.com/urigo/graphql-cli#migration-from-3xx-to-4xx) and we will be happy to hear your feedback during the alpha period!

Feel free to contact us by opening an issue or using our Discord channel;
https://discord.gg/xud7bH
